### ISSUE TITLE
Examples for stream-room-messages and stream-livechat-room are interchanged

### ISSUE THREAD

**Issue by mhutter** [RocketChat/developer-docs#154]
See the first code blocks on 

https://developer.rocket.chat/reference/api/realtime-api/subscriptions/stream-room-messages

and

https://developer.rocket.chat/reference/api/realtime-api/livechat-api/streamlivechatroom

I don't know if only the subscription `name` are wrong or the rest of the code as well.

**Comment by mhutter** [comment]
Introduced in #77 and cfd900e7b326a197bf852d540cb8ee772be25dd9

**Comment by SJ-Kumar** [comment]
Please let me know if I can start work on this task.

**Comment by varunsood2003** [comment]
hey could you please explain exactly what is the issue ?

**Comment by SJ-Kumar** [comment]
> @SJ-Kumar Yes, you can start working on it, please.

Sorry for the delayed response. Is it just the word interchange or we should debug and check if even the content in the code has to tweaked. Please let me know.

**Comment by SJ-Kumar** [comment]
For **stream-livechat-room** usage, _visitorToken_ usage seems to be correct.

**Comment by mhutter** [comment]
> hey could you please explain exactly what is the issue ?

The example DDP Messages on

https://developer.rocket.chat/reference/api/realtime-api/subscriptions/stream-room-messages

and

https://developer.rocket.chat/reference/api/realtime-api/livechat-api/streamlivechatroom

are wrong. They do not use the correct message type.

**Comment by SJ-Kumar** [comment]
Merged the updated page by creating a pull request.

**Comment by mhutter** [comment]
The linked PR (#178) does not fix the fact that the example still uses the wrong message type

**Comment by mhutter** [comment]
Let me rephrase the problem again.

If you go to https://developer.rocket.chat/reference/api/realtime-api/subscriptions/stream-room-messages, the title says "stream-room-messages".

However, the example code block has a message of a different type:

    ...
    "name":"stream-livechat-room",
    ...

Since this field is wrong, I suspect that the rest of the example is ALSO wrong. However, since this document is supposed to be the official reference, I cannot verify this without reading through the appropriate source code.

Since I am not familiar with RocketChat's code base, one of the maintainers please check the actual implementation and update the documentation to properly document ALL available fields, possible values, constraints, ... etc.

The same issue also exists for https://developer.rocket.chat/reference/api/realtime-api/livechat-api/streamlivechatroom (and possibly other pages).


---

As a 3rd party developer,
In order to develop working integrations
And not having to reverse-engineer Rocket Chat
I'd like to have an accurate & up-to-date description of its interfaces.

**Comment by rachana-visavadiya** [comment]
The titles and the names in the example blocks match. The example blocks are also correct for the mentioned pages.
